### PR TITLE
Modernize built-in components (part 2)

### PR DESCRIPTION
**Pull request by chancancode**
[Rendered](https://github.com/emberjs/rfcs/blob/modernizing-built-in-components/text/0707-modernize-built-in-components-2.md)

### CODE REVIEW

File: text/0707-modernize-built-in-components-2.md [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+  * `@selectionEnd`
+  * `@selectionStart`
+  * `@spellcheck`
+  * `@tabindex`
+  * `@title`
+  * `@wrap`
+
+These arguments are no longer necessary – with angle bracket invocations, HTML
+attributes can be passed directly. An invocation passing one or more of these
+named arguments shall trigger a deprecation warning simialr to this:
```
Comment (CvX) [inline text/0707-modernize-built-in-components-2.md:180]:
```suggestion
named arguments shall trigger a deprecation warning similar to this:
```

File: text/0707-modernize-built-in-components-2.md [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+  * `@focus-in` (for the `focusin` event)
+  * `@focus-out` (for the `focusout` event)
+  * `@key-down` (for the `keydown` event)
+  * `@key-press` (for the `keypress` event)
+  * `@key-up` (for the `keyup` event)
+
+These arguments are no longer necessary – with angle bracket invocations, DOM
+event listeners can be registered directly using the `{{on}}` modifier. An
+invocation passing one or more of these named arguments shall trigger a
+deprecation warning simialr to this:
```
Comment (CvX) [inline text/0707-modernize-built-in-components-2.md:291]:
```suggestion
deprecation warning similar to this:
```

File: text/0707-modernize-built-in-components-2.md [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+or
+
+{{#link-to tagName="div" ...}}...{{/link-to}}
+           ~~~~~~~~~~~~~
+
+Passing the `@tagName` argument to <LinkTo> is deprecated. Using a <div>
+element for navigation is not recommended as it creates issues with assistive
+technologies. Remove this argument to use the default <a> element. In the rare
+cases that calls for using a different element, refactor to use the router
+service inside a custom event handler instead.
```
Comment (ro0gr) [inline text/0707-modernize-built-in-components-2.md:387]: Does it make sense to have a link here to appropriate section of the RRF, to provide more context, and detailed description like it is in the RFC.

This "no longer recommended" feels like a soft form of deprecations. But deprecations typically have their link.

**Comment by BryanCrotaz** [comment]
Rendered link is a 404?

**Comment by chancancode** [comment]
> Rendered link is a 404?

Fixed now. Generally this is a problem after merging the RFCs because while still in PRs, the rendered links points to the branch, but once it's merged the branch is deleted and it lives on the master branch instead, so the link would have to be updated

**Pull request merged**